knative-operator/hack/009-generated-job-name.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/knative-operator/deploy/resources/knativekafka/3-channel-post-install.yaml b/knative-operator/deploy/resources/knativekafka/3-channel-post-install.yaml
+index c316076c..8258531c 100644
+--- a/knative-operator/deploy/resources/knativekafka/3-channel-post-install.yaml
++++ b/knative-operator/deploy/resources/knativekafka/3-channel-post-install.yaml
+@@ -162,7 +162,7 @@ roleRef:
+ apiVersion: batch/v1
+ kind: Job
+ metadata:
+-  name: eventing-kafka-v0.26-channel-post-install-job
++  name: v0.26-eventing-kafka-channel-post-install-job
```

Comment:
Using the 'old' semantics for "update" images.

Starting with 1.21 (or upstream 0.27/1.0) we will have `generatedName`...  see the `update-manifest` for the actual comment

Reply:
Should we patch towards the "future" version, i.e. make this patch to the `generatedName`: `storage-version-migration-eventing-kafka-channel-` as done in https://github.com/knative-sandbox/eventing-kafka/blob/79f02ebad7dc2fa9117236d85d06fb83b36c2fa0/config/channel/post-install/storage-version-migrator.yaml#L18?

We'd then not need to change the csv.sh etc. again and just drop the patch in 1.0.

Reply:
yeah, but next release (1.0) we have a new post-install for the source too. Than they all are `generatedName`, I did that upstream - and than I think I am generally touching them, so will do it than - for all

Reply:
alrighty